projects/packages/backup/src/js/components/back-up-now/index.jsx

```diff
@@ -0,0 +1,114 @@
+import { Button } from '@automattic/jetpack-components';
+import apiFetch from '@wordpress/api-fetch';
+import { Tooltip } from '@wordpress/components';
+import { useSelect } from '@wordpress/data';
+import PropTypes from 'prop-types';
+import { useCallback, useEffect, useState } from 'react';
+import { BACKUP_STATE } from '../../constants';
+import useAnalytics from '../../hooks/useAnalytics';
+import useBackupsState from '../../hooks/useBackupsState.js';
+import { STORE_ID } from '../../store';
+
+export const BackupNowButton = ( {
+	children,
+	tooltipText,
+	tracksEventName,
+	variant = 'primary',
+	weight = 'regular',
+	onClick,
+} ) => {
+	const { tracks } = useAnalytics();
+	const [ buttonContent, setButtonContent ] = useState( children );
+	const [ currentTooltip, setCurrentTooltip ] = useState( tooltipText );
+	const [ isEnqueuing, setIsEnqueuing ] = useState( false );
+	const [ enqueued, setEnqueued ] = useState( false );
+	const areBackupsStopped = useSelect( select => select( STORE_ID ).getBackupStoppedFlag() );
+	const { backupState, fetchBackupsState } = useBackupsState( enqueued );
+	const backupCurrentlyInProgress = backupState === BACKUP_STATE.IN_PROGRESS;
+	const disabled = isEnqueuing || enqueued || backupCurrentlyInProgress || areBackupsStopped;
+	const onClickHandler = useCallback(
+		event => {
+			if ( tracksEventName ) {
+				tracks.recordEvent( tracksEventName );
+			}
+
+			setIsEnqueuing( true );
+
+			apiFetch( { method: 'POST', path: `/jetpack/v4/site/backup/enqueue` } ).then( () => {
+				setIsEnqueuing( false );
+				setEnqueued( true );
+				fetchBackupsState();
+			} );
+
+			if ( onClick ) {
+				onClick( event );
+			}
+		},
+		[ tracksEventName, onClick, tracks, fetchBackupsState ]
+	);
+
+	useEffect( () => {
+		const statusLabels = {
+			QUEUEING: 'Queueing backup',
```

Comment:
Need translations for all these states and the 'Back up now' passed to the component

Reply:
Good catch. I just added the translations! In the labels and tooltip descriptions, including the `Back up now`.